src/states/elder/mod.rs

```diff
@@ -1321,6 +1321,29 @@ impl Base for Elder {
         Transition::Stay
     }
 
+    fn handle_connection_failure(
+        &mut self,
+        addr: SocketAddr,
+        _outbox: &mut dyn EventBox,
+    ) -> Transition {
```

Comment:
Commit: Send Ping message to trigger lost detection on connection failure 

Should that happen for Adults as well? i.e to detect their Elders are gone? Or maybe we don't really care in this case I guess.

Reply:
No because adults currently don't do anything when a peer gets lost.

Reply:
Being informed by other Elders shall be enough for Adults?